src/structs/notifications.rs

```diff
@@ -44,6 +32,34 @@ pub struct PacketsNotification {
     pub previous_threshold: u32,
 }
 
+impl Default for PacketsNotification {
+    fn default() -> Self {
+        PacketsNotification {
+            threshold: None,
+            sound: Sound::Gulp,
+            previous_threshold: 750,
+        }
+    }
+}
+
+impl PacketsNotification {
+    /// Arbitrary string constructor. Will fallback values to existing notification if set, default() otherwise
+    pub fn from(value: &str, existing: Option<Self>) -> Self {
```

Comment:
This function content should be the same of the respective previous segment of code in settings.rs (e.g, when text field is empty, consider it as a zero and not as a None; I'd like this functions to have the same content and behaviour as the previous version of the code)